cmd/PublishGrantEvents/handler.go

```diff
@@ -112,7 +112,7 @@ func buildGrantModificationEventJSON(record events.DynamoDBEventRecord) ([]byte,
 		}
 		if err := prevVersion.Validate(); err != nil {
 			sendMetric("grant_data.invalid", 1, metricTag)
-			log.Warn(logger, "grant data from ItemMapper is invalid", err)
+			log.Warn(logger, "grant data from ItemMapper is invalid", "error", err)
```

Comment:
FYI: This change is wholly unrelated to the rest of this PR; I just happened to notice that we were missing the `error` key for this log in Datadog (odd numbers of k/v arguments make things weird). We can probably lint for this in the future.